node/src/chains/common.rs

```diff
@@ -119,6 +120,8 @@ pub fn runtime_genesis_config(
 		balances: BalancesConfig { balances },
 		staking: StakingConfig {
 			stakers,
+			validator_count,
+			minimum_validator_count: 1,
```

Comment:
minimum validator count doesn't play a big role in the overall working of the chain. The issue with it is that it can cause the network to get soft-bricked (just like what happened  to Dymension) if there are not enough validators to meet the minimum.